src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -559,14 +570,31 @@ export default defineComponent({
       this.$store.commit('addVideoToDeArrowCache', cacheData)
 
       // fetch dearrow thumbnails if enabled
-      if (this.useDeArrowThumbnails && this.deArrowCache?.thumbnail === null) {
+      if (this.showDeArrowThumbnail && this.deArrowCache?.thumbnail === null) {
         if (this.debounceGetDeArrowThumbnail == null) {
           this.debounceGetDeArrowThumbnail = debounce(this.fetchDeArrowThumbnail, 1000)
         }
 
         this.debounceGetDeArrowThumbnail()
       }
     },
+    toggleDeArrow() {
+      if (!this.deArrowChangedContent) {
+        return
+      }
+
+      this.deArrowTogglePinned = !this.deArrowTogglePinned
+      this.deArrowToggleTitle = this.deArrowTogglePinned
+        ? this.$t('Video.DeArrow.Show Modified Details')
+        : this.$t('Video.DeArrow.Show Original Details')
```

Comment:
Please create a computed prop for this instead, that way when the user changes the language the text will update automatically, whereas at the moment you would have to manually click the button for it to switch to the correct language. The use of translations strings in FreeTube should be reactive wherever possible.